tests/php/test-tag-and-attribute-sanitizer.php

```diff
@@ -2234,8 +2234,8 @@ static function() {
 				[ AMP_Tag_And_Attribute_Sanitizer::MISSING_URL, AMP_Tag_And_Attribute_Sanitizer::ATTR_REQUIRED_BUT_MISSING ],
 			],
 
-			'amp_img_missing_url'                          => [
-				'<amp-img src="" height="100" width="200"></amp-img>',
```

Comment:
Huh? Was this already in there as a duplicate?

Reply:
Yes, it was a duplicate. 

> ![image](https://user-images.githubusercontent.com/134745/93363065-9791fb00-f7fb-11ea-9e97-11100f8ec4b2.png)

So I changed it to just check the `srcset` instead of the `src`.